### PR TITLE
:sparkles: add certification results in search page

### PR DESCRIPTION
**Pull request by benjaminbrgs**
Closes https://github.com/Zenika/skillZ/issues/534

### CODE REVIEW

File: src/components/organisms/CertificationPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          </div>
+        </div>
+        <div
+          className="flex w-1/6 justify-end cursor-pointer"
+          onClick={(e) => {
+            e.stopPropagation();
+            push(linkToCertification);
+          }}
+        >
+          <AiFillEye
```
Comment (MailyLehoux) [inline src/components/organisms/CertificationPanel.tsx:37]: I think we should use same way than the skill panels : 

Capture d’écran de 2023-02-15 10-39-16

With the logo and the number of user. And by the way, I would suggest to click on this button, for the skill panel as well, to access to the user panel. And so, delete the eye. What do you think about that ?

Comment (benjaminbrgs) [reply src/components/organisms/CertificationPanel.tsx]: Yes, you're right! I've just committed the changes

File: src/components/organisms/CertificationPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const { push } = useRouter();
+
+  const linkToCertification = new URL(
+    `${config.nextPublicBaseUrl}/certifications/${encodeURIComponent(name)}`
+  );
+
+  return (
+    <>
+      <div
+        className={`flex flex-row bg-light-light dark:bg-dark-light px-4 py-4 mx-2 my-1 rounded-lg items-center cursor-pointer border border-light-light dark:border-dark-light hover:bg-light-dark hover:border-light-graybutton hover:dark:bg-dark-radargrid hover:dark:border-dark-graybutton`}
```
Comment (MailyLehoux) [inline src/components/organisms/CertificationPanel.tsx:21]:
```suggestion
        className={`flex flex-row bg-light-light dark:bg-dark-light px-4 py-4 mx-2 my-1 rounded-lg items-center border border-light-light dark:border-dark-light`}
```

**Review by MailyLehoux — APPROVED** [review]
:+1: its works nice. But maybe we should delete hover and cursor if we cannot click on it

**Pull request merged**